configure.ac

```diff
@@ -114,10 +115,13 @@ AC_DEFUN([CH_CHECK_VERSION], [
     vact=$($prog $3)
     AX_COMPARE_VERSION([$2], [le], [$vact],
       [AC_MSG_RESULT([ok ($vact)])
-       AC_SUBST([$1_VERSION], [$vact])],
+       AC_SUBST([$1_VERSION], [$vact]),
```

Comment:
The proposal was to remove `FOO_VERSION`. Would you like to argue to keep it?

Reply:
Ah. Sorry. `FOO_VERSION` is used by all of the AS_IF lines 
https://github.com/hpc/charliecloud/blob/03f546c7bbd19cdf9ef0a9f96761e37ea6b9119c/configure.ac#L342. 
Can that be done with the version note string? Cause it seems like it expects just a version number and the note has the message as well.

Reply:
Figured out how to do this properly with `FOO_VERSION_NOTE`.

Reply:
`AS_IF` can all use plain `$FOO` I think.

Reply:
I modified it from `AS_IF([test -n "$NVIDIA_CLI_VERSION"]` to `AS_IF([test  "$NVIDIA_CLI_VERSION" != "not found"]`